apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/hooks/use-site.ts

```diff
@@ -21,7 +21,7 @@ export function useSite() {
 	);
 
 	return {
-		isFreePlan: site?.plan?.is_free,
+		isPaidPlan: site && ! site.plan.is_free,
```

Comment:
Note that the `?` was added in plan in https://github.com/Automattic/wp-calypso/pull/44895

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/44909/commits/b5a84b06ad7be39e4d863e6fb7f7c3ef63f45c5d